app_feup/lib/view/Pages/login_page_view.dart

```diff
@@ -200,6 +200,8 @@ class _LoginPageViewState extends State<LoginPageView> {
   /// Creates the widget for the user to keep signed in (save his data).
   Widget createSaveDataCheckBox() {
     return CheckboxListTile(
+      activeColor: Colors.white,
+      checkColor: Theme.of(context).primaryColor,
```

Comment:
I think the login page is a very specific case where the logic of theme is different from the whole rest of the app. I'm not sure if it makes sense to create a separate group of variables just to accommodate for that, instead of directly changing the properties in the page widgets themselves.

Reply:
I agree with @bdmendes. It's just a case where this happens, so we can keep the logic here.
Maybe at some point in the future, when needed, we can change that.